crates/bevy_core_pipeline/src/deferred/mod.rs

```diff
@@ -88,36 +92,21 @@ impl CachedRenderPipelinePhaseItem for Opaque3dDeferred {
 ///
 /// Used to render all meshes with a material with an alpha mask.
 pub struct AlphaMask3dDeferred {
-    pub asset_id: AssetId<Mesh>,
-    pub entity: Entity,
-    pub pipeline_id: CachedRenderPipelineId,
-    pub draw_function: DrawFunctionId,
+    pub key: Opaque3dPrepassBinKey,
```

Comment:
Should we generalize the name of this so that it's not tied to the Opaque3dPrepass?

Reply:
I renamed it to `OpaqueNoLightmap3dBinKey`.